cmd/simulateEvol/genic.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/expandedTree"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/simulate"
+	"log"
+	"math/rand"
+	"os"
+)
+
+type GenicSettings struct {
+	TreeFile     string
+	InFile       string
+	OutFile      string
+	SetSeed      int64
+	SimOutFile   string
+	GenePredFile string
+}
+
+func GenicUsage(genicFlags *flag.FlagSet) {
+	fmt.Print(
+		"simulateEvol genic - a tool to simulate molecular evolution in coding regions, using the BLOSOM matrix.\n" +
+			"Usage:\n" +
+			"simulateEvol genic tree.newick in.fasta out.fasta\n" +
+			"options:\n")
+	genicFlags.PrintDefaults()
+}
+
+func parseGenicArgs() {
```

Comment:
I am a little confused about this concept of flags, how the function parseGenicArgs works... Is it is a function designed to handle the parsing of command-line arguments for the simulateEvol genic tool, in terms of words?